sql/2_0_0-to-2_0_1_upgrade.sql

```diff
@@ -164,11 +164,11 @@ CREATE TABLE `updater_user_mode_backup_entry` (
 
 --Appointment Cancellation Reasons
 #IfNotRow2D list_options list_id lists option_id cancellation_reasons
-INSERT INTO list_options ( list_id, option_id, title,activity ) VALUES ('lists','cancellation_reasons','Cancellation Reasons', 1);
-INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES ('cancellation_reasons','1','No reason given',5,0);
-INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES ('cancellation_reasons','2','Work',10,0);
-INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES ('cancellation_reasons','3','Sick',20,0);
-INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES ('cancellation_reasons','4','Weather',25,0);
+INSERT INTO `list_options` ( `list_id`, `option_id`, `title`,`activity` ) VALUES ('lists','cancellation_reasons','Cancellation Reasons', 1);
```

Comment:
I think this doesn;t do some extra work rather than making the code cleaner. Looks ok to me. Good @Mangielijah , @teryhill your opinion

Reply:
@Mangielijah  what are you fixing in this PR. The back ticks are not required.